README.md

```diff
@@ -21,635 +22,22 @@ variety of helper functions and patterns for common infrastructure testing tasks
 - Running shell commands
 - And much more
 
-For an introduction to Terratest, including unit tests, integration tests, end-to-end tests, dependency injection, test 
-parallelism, retries, error handling, and static analysis, see the talk [Automated Testing for Terraform, Docker, 
+For an introduction to Terratest, including unit tests, integration tests, end-to-end tests, dependency injection, test
+parallelism, retries, error handling, and static analysis, see the talk [Automated Testing for Terraform, Docker,
 Packer, Kubernetes, and More](https://www.infoq.com/presentations/automated-testing-terraform-docker-packer/) and the
-blog post [Open sourcing Terratest: a swiss army knife for testing infrastructure 
+blog post [Open sourcing Terratest: a swiss army knife for testing infrastructure
```

Comment:
Can remove this paragraph, as the website will handle it

Reply:
Sure!